lua/telescope/previewers.lua

```diff
@@ -55,6 +57,18 @@ function Previewer:teardown()
   end
 end
 
+function Previewer:send_input(input)
+  if self._send_input then
```

Comment:
if we don't have `_send_input`, we should display some error to the user (so that they know why it isn't working)

Reply:
Done in the latest commit:)